src/game/level_update.c

```diff
@@ -552,7 +552,7 @@ s16 music_unchanged_through_warp(s16 arg) {
     s16 unchanged = TRUE;
     s16 currBgMusic;
```

Comment:
`currBgMusic` and `levelNum` are unused and will appropriately throw warnings here, can you move these declarations inside of the ifdef?

Reply:
Hopefully fixed this in my latest commit, thank you for the heads up!